src/generator.js

```diff
@@ -66,11 +81,25 @@ const genCitation = (citeproc, entries, citationId, citationPre, properties = {
  * @param {*} citeproc
  */
 const genBiblioNode = (citeproc) => {
-  const [, bibBody] = citeproc.makeBibliography()
+  const [params, bibBody] = citeproc.makeBibliography()
   const bibliography =
     '<div id="refs" class="references csl-bib-body">\n' + bibBody.join('') + '</div>'
   const biblioNode = htmlToHast(bibliography)
-  return biblioNode
+
+  // Add citekey id to each bibliography entry.
+  const biblioEntries = {}
+  biblioNode.children
+    .filter((node) => node.properties?.className?.includes('csl-entry'))
+    .forEach((node, i) => {
+      const citekey = params.entry_ids[i][0].toLowerCase()
+
+      node.properties = node.properties || {}
+      node.properties.id = 'bib-' + citekey
```

Comment:
Similar to the citation id, we should include the `citationId` which might have to disambiguate citations with same name and date. Would also make it easier to match citation and bib.

Reply:
How would I get the `citationID` in this case? Are there any options to pass into `citeproc.makeBibliography` or other functions to get that info?

Reply:
Isn't the citekey unique? That's why I didn't include the `citationID` here.

Reply:
I mixed up the two. `citationID` is for an inline citation, `citekey` is for a citation.